go.mod

```diff
@@ -6,50 +6,61 @@ require (
 	github.com/onsi/ginkgo/v2 v2.9.2
 	github.com/onsi/gomega v1.27.6
 	github.com/pkg/errors v0.9.1
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
-	k8s.io/client-go v0.26.4
+	k8s.io/api v0.27.1
+	k8s.io/apimachinery v0.27.1
+	k8s.io/client-go v0.27.1
```

Comment:
All our other projects are still pointing to K8s 1.26 (v0.26.x), I'm wondering if it's wise to be the first to require 1.27

Reply:
The other projects will soon go to 1.27 as soon as a new controller-runtime tag is published which is imminent.